<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="33997" started-at="2017-09-04T18:00:24Z" finished-at="2017-09-04T18:00:58Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="33997" started-at="2017-09-04T18:00:24Z" finished-at="2017-09-04T18:00:58Z">
      <class name="com.softserve.edu.FirefoxProfilesTest">
        <test-method status="PASS" signature="startDriver()[pri:0, instance:com.softserve.edu.FirefoxProfilesTest@215be6bb]" name="startDriver" is-config="true" duration-ms="15" started-at="2017-09-04T21:00:24Z" finished-at="2017-09-04T21:00:24Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startDriver -->
        <test-method status="FAIL" signature="ourSettingsForFireFox()[pri:0, instance:com.softserve.edu.FirefoxProfilesTest@215be6bb]" name="ourSettingsForFireFox" duration-ms="33949" started-at="2017-09-04T21:00:24Z" finished-at="2017-09-04T21:00:58Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Cheese! - ÐŸÐ¾ÑˆÑƒÐº Google] but found [Cheese! - Пошук Google]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Cheese! - ÐŸÐ¾ÑˆÑƒÐº Google] but found [Cheese! - Пошук Google]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at org.testng.Assert.assertEquals(Assert.java:199)
	at com.softserve.edu.FirefoxProfilesTest.ourSettingsForFireFox(FirefoxProfilesTest.java:124)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ourSettingsForFireFox -->
      </class> <!-- com.softserve.edu.FirefoxProfilesTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
